bluepyopt/ephys/efeatures.py

```diff
@@ -22,6 +22,7 @@
 
 import logging
 
+from bluepyopt.ephys.serializer import DictMixin
```

Comment:
Mmm, I guess we can leave it here for now, probably fairly easy to change afterwards. I would be fairly 'cool' if we can read/write and entire optimisation configuration this way